schema.pgsql

```diff
@@ -16,14 +16,61 @@ CREATE INDEX files_expiry ON files(expiry);
 CREATE TABLE release_versions (
     project varchar(50) PRIMARY KEY,
     version varchar(25) NOT NULL,
+    prerelease_version varchar(25),
     updated TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW()
 );
 
 -- Abritrary version values at the (approx) time this was written; this don't really matter as
 -- they'll get updated within a first few seconds of initial startup.
-INSERT INTO release_versions (project, version, updated) VALUES ('oxen-io/session-desktop', 'v1.7.3', '2021-10-14Z');
-INSERT INTO release_versions (project, version, updated) VALUES ('oxen-io/session-android', '1.11.11', '2021-10-14Z');
-INSERT INTO release_versions (project, version, updated) VALUES ('oxen-io/session-ios', '1.11.17', '2021-10-14Z');
+INSERT INTO release_versions (project, version, prerelease_version, updated) VALUES ('oxen-io/session-desktop', 'v1.7.3', NULL, '2021-10-14Z');
+INSERT INTO release_versions (project, version, prerelease_version, updated) VALUES ('oxen-io/session-android', '1.11.11', NULL, '2021-10-14Z');
+INSERT INTO release_versions (project, version, prerelease_version, updated) VALUES ('oxen-io/session-ios', '1.11.17', NULL, '2021-10-14Z');
+
+CREATE TABLE release_notes (
+    project varchar(50) NOT NULL,
+    version varchar(25) NOT NULL,
+    name TEXT,
+    notes TEXT
+);
+
+CREATE INDEX release_notes_project_version ON release_notes(project, version);
+
+CREATE TABLE release_assets (
+    project varchar(50) NOT NULL,
+    version varchar(25) NOT NULL,
+    name varchar(225) NOT NULL,
+    url varchar(225) NOT NULL
+);
+
+CREATE INDEX release_assets_project_version ON release_assets(project, version);
+
+INSERT INTO release_assets (project, version, name, url) VALUES ('oxen-io/session-desktop', 'v1.7.3', 'Test name', 'github.com');
+
+CREATE TABLE account_version_checks (
+    blinded_id varchar(66) NOT NULL,
+    platform varchar(25) NOT NULL,
+    timestamp TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    CONSTRAINT unique_blinded_id_platform_timestamp UNIQUE (blinded_id, platform, timestamp)
+);
+
+CREATE INDEX account_version_checks_blinded_id ON account_version_checks(blinded_id);
+
+CREATE TABLE session_token_stats (
+    maximum_supply INT NOT NULL,
+    sent_per_node INT NOT NULL,
+    staking_reward_pool INT NOT NULL,
+);
+CREATE TABLE session_token_history (
+    current_value NUMERIC(20, 6) NOT NULL,
+    circulating_supply INT NOT NULL,
+    total_nodes INT NOT NULL,
+    updated TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW()
+);
+
+-- Abritrary values at the (approx) time this was written; this don't really matter as
+-- they'll get updated within a first few seconds of initial startup.
+INSERT INTO session_token_stats (maximum_supply, sent_per_node, staking_reward_pool) VALUES (240000000, 15000, 40000000);
+INSERT INTO session_token_history (current_value, circulating_supply, total_nodes, updated) VALUES (0.099002, 30400000, 2194, 68297852, '2024-06-14Z');
```

Comment:
So these tables are used for the new `/token_info` endpoint (which we have in-progress designs for - I think I need to make minor changes to the API/structure based on the latest designs anyway)

For this one I was asked to stub-out an endpoint to populate the screen (and do as much work in advance as I could) but, since the token doesn't exist yet, creating a script to populate these tables wasn't really something I could do - I was envisioning something like what we have in `cleanup.py` which just fetches token stats once per day from somewhere and stores them in here

Reply:
These values are actually also test data that I should have removed - will remove them from the next commit

I'm just wondering whether it'd be better for the `/token_info` to just return 0-ed values if there is nothing in the database (at the moment it returns a `502` error) 🤔